TodoList/View Controllers/TodoListViewController.swift

```diff
@@ -13,6 +13,14 @@ class TodoListViewController: UITableViewController {
         return footer
     }()
 
+    private lazy var editBarButtonItem: UIBarButtonItem = {
+        let item = UIBarButtonItem(title: nil,
+                                   style: .plain,
+                                   target: self,
+                                   action: #selector(didTapEditBarButtonItem(_:)))
+        return item
+    }()
```

Comment:
I'm not sure if there's any gotcha but did you know about this? https://developer.apple.com/documentation/uikit/uiviewcontroller/1621471-editbuttonitem

It's localized and it also changes title based on editing state. It would save yourself a bunch of code. Would you mind giving it a try?

Reply:
Holy, that's awesome. Did not have a clue that exists :)
Thanks, I' ll definitely try it.

So make changes and then push to the same branch right?

Reply:
Yeah, work on the same branch until it's approved and merged into master. Please don't forget to post the gif. This is the app I told you about: https://itunes.apple.com/us/app/giphy-capture.-the-gif-maker/id668208984?mt=12